src/core/Akka.NodeTestRunner/Akka.NodeTestRunner.csproj

```diff
@@ -5,6 +5,7 @@
     <TargetFrameworks>$(NetFrameworkTestVersion);$(NetTestVersion);$(NetCoreTestVersion)</TargetFrameworks>
     <OutputType>Exe</OutputType>
     <RuntimeIdentifiers>win10-x64</RuntimeIdentifiers>
+    <NoWarn>NU1605</NoWarn>
```

Comment:
Can't compile without this - the `xunit.runner.utilities` package is ~3 years old and brings in a bunch of .NET Standard 1.6 dependencies that clash with the new .NET Standard 2.0 ones DotNetty includes.